### ISSUE TITLE
Uncaught TypeError: Cannot read property 'getPath' of null

### ISSUE THREAD

**Issue by Bolza** [atom/tree-view#295]
[Enter steps to reproduce below:]

1. ...
2. ...

**Atom Version**: 0.165.0
**System**: linux 3.2.0-74-generic
**Thrown From**: [tree-view](https://github.com/atom/tree-view) package, v0.139.0

### Stack Trace

Uncaught TypeError: Cannot read property 'getPath' of null

```
At /usr/share/atom/resources/app/node_modules/tree-view/lib/tree-view.js:876

TypeError: Cannot read property 'getPath' of null
  at TreeView.module.exports.TreeView.add (/usr/share/atom/resources/app/node_modules/tree-view/lib/tree-view.js:876:35)
  at atom-workspace.disposables.add.atom.commands.add.tree-view:add-file (/usr/share/atom/resources/app/node_modules/tree-view/lib/main.js:64:39)
  at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/usr/share/atom/resources/app/src/command-registry.js:243:29)
  at /usr/share/atom/resources/app/src/command-registry.js:3:61
  at CommandPaletteView.module.exports.CommandPaletteView.confirmed (/usr/share/atom/resources/app/node_modules/command-palette/lib/command-palette-view.js:120:32)
  at CommandPaletteView.module.exports.SelectListView.confirmSelection (/usr/share/atom/resources/app/node_modules/autocomplete/node_modules/atom-space-pen-views/lib/select-list-view.js:335:21)
  at space-pen-div.atom.commands.add.core:confirm (/usr/share/atom/resources/app/node_modules/autocomplete/node_modules/atom-space-pen-views/lib/select-list-view.js:109:19)
  at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/usr/share/atom/resources/app/src/command-registry.js:243:29)
  at /usr/share/atom/resources/app/src/command-registry.js:3:61
  at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/usr/share/atom/resources/app/node_modules/atom-keymap/lib/keymap-manager.js:549:16)
  at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/usr/share/atom/resources/app/node_modules/atom-keymap/lib/keymap-manager.js:391:22)
  at HTMLDocument.module.exports.WindowEventHandler.onKeydown (/usr/share/atom/resources/app/src/window-event-handler.js:167:20)

```

### Commands

```
  3x -0:56.0 project-find:show (input.hidden-input)
     -0:54.2 editor:consolidate-selections (input.hidden-input)
     -0:54.2 core:cancel (input.hidden-input)
     -0:52.4 command-palette:toggle (input.hidden-input)
 14x -0:48.9 core:move-down (input.hidden-input)
  2x -0:45.8 core:move-up (input.hidden-input)
     -0:45.0 core:move-down (input.hidden-input)
  2x -0:43.9 core:move-up (input.hidden-input)
 12x -0:43.2 core:move-down (input.hidden-input)
     -0:24.8 core:move-up (input.hidden-input)
  3x -0:24.2 core:move-down (input.hidden-input)
  5x -0:23.0 core:backspace (input.hidden-input)
  5x -0:16.2 core:move-down (input.hidden-input)
 15x -0:12.9 core:backspace (input.hidden-input)
     -0:00.0 core:confirm (input.hidden-input)
     -0:00.0 tree-view:add-file (input.hidden-input)
```

### Config

```
{
  "core": {
    "autoHideMenuBar": true
  }
}
```

### Installed Packages

```coffee
# User
No installed packages

# Dev
No dev packages
```

/cc @atom/core

**Comment by tralamazza** [comment]
+1

**Comment by kevinsawicki** [comment]
This is a duplicate of https://github.com/atom/tree-view/issues/273 and should be fixed in the 0.166 release.
